### PR TITLE
Get rid of mobile Kepler class and add missing Kepler GPUs

### PR DESCRIPTION
**Pull request by xddxd**
Not needed anymore since async compute is disabled for every Kepler GPU. Also adds missing IDs for GK210 (Tesla K80 and such).

### CODE REVIEW

File: rpcs3/Emu/RSX/VK/vkutils/chip_class.cpp [hunk 1]
```diff
@@ -33,30 +33,22 @@ namespace vk
...
 		table.add(0x1401, 0x1427, chip_class::NV_maxwell); // GM206
 		table.add(0x15F7, 0x15F9, chip_class::NV_pascal);  // GP100 (Tesla P100)
 		table.add(0x1B00, 0x1D80, chip_class::NV_pascal);
 		table.add(0x1D81, 0x1DBA, chip_class::NV_volta);
-		table.add(0x1E02, 0x1F54, chip_class::NV_turing);  // TU102, TU104, TU106, TU106M, TU106GL (RTX 20 series)
-		table.add(0x1F82, 0x1FB9, chip_class::NV_turing);  // TU117, TU117M, TU117GL
-		table.add(0x2182, 0x21D1, chip_class::NV_turing);  // TU116, TU116M, TU116GL
-		table.add(0x20B0, 0x20BE, chip_class::NV_ampere);  // GA100
-		table.add(0x2204, 0x25AF, chip_class::NV_ampere);  // GA10x (RTX 30 series)
+		table.add(0x1E02, 0x1F54, chip_class::NV_turing); // TU102, TU104, TU106, TU106M, TU106GL (RTX 20 series)
```
Comment (kd-11) [inline rpcs3/Emu/RSX/VK/vkutils/chip_class.cpp:47]: Spaces here kept the comments aligned vertically. Not a big deal but unnecessary change nonetheless.

Comment (xddxd) [reply rpcs3/Emu/RSX/VK/vkutils/chip_class.cpp:47]: Oops, unintentional.

**Comment by Augusto7743** [comment]
You are saying the option in GPU setings Shader Mode "async with sahder interpreter" not work with Kepler GPUs ?
